packages/react-native-codegen/src/parsers/flow/components/props.js

```diff
@@ -58,11 +56,13 @@ function buildPropSchema(
 function extendsForProp(
   prop: PropAST,
   types: TypeDeclarationMap,
+  parser: Parser,
 ): ExtendsForProp {
-  if (!prop.argument) {
+  const argument = parser.argumentForProp(prop);
+  if (argument) {
```

Comment:
This now does exactly the opposite of what it used/supposed to do :)

It should instead be:
```
if (!argument) {
```

Otherwise this would break codegen completely, at least on our internal tests.

I am surprised, though, that CI on Github didn't catch this (the only failed test seems inside the app template and is unrelated).

I think we need a follow-up on that, CC @cipolleschi 

@siddarthkay - note that I fixed this internally myself this time, to save one extra back and forth, but normally that would be your responsibility :)

Reply:
>I am surprised, though, that CI on Github didn't catch this (the only failed test seems inside the app template and is unrelated).

I think, those files don't have tests, but, other tests should have been failed; or may be the current test-suite _doesn't_ test this specific case.